recipes/openexr/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+
+
+class OpenEXRConan(ConanFile):
+    name = "openexr"
+    description = "OpenEXR is a high dynamic-range (HDR) image file format developed by Industrial Light & " \
+                  "Magic for use in computer imaging applications."
+    license = "BSD"
+    homepage = "https://github.com/openexr/openexr"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "namespace_versioning": [True, False], "fPIC": [True, False]}
+    default_options = "shared=False", "namespace_versioning=True", "fPIC=True"
+    generators = "cmake"
+    exports_sources = ["IlmImf__ImfSystemSpecific.cpp.patch"]
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.remove("fPIC")
+
+    def requirements(self):
+        self.requires('zlib/1.2.11')
+
+    def configure(self):
+        if "fPIC" in self.options.fields and self.options.shared:
+            self.options.fPIC = True
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        tools.replace_in_file(os.path.join('openexr-{}'.format(self.version), 'CMakeLists.txt'),
+                              'project(OpenEXR VERSION ${OPENEXR_VERSION})',
+                              """project(OpenEXR VERSION ${OPENEXR_VERSION})
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(KEEP_RPATHS)
+
+set(OPENEXR_PACKAGE_PREFIX ${CMAKE_INSTALL_PREFIX})
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+""")
+        for p in self.conan_data["patches"][self.version]:
+            tools.patch(**p)
+
+    def build(self):
+        # parallel builds are disabled due to the random issue: 'toFloat.h': No such file or directory
+        cmake = CMake(self, parallel=self.settings.os != 'Windows')
+        cmake.definitions["OPENEXR_BUILD_PYTHON_LIBS"] = False
+        cmake.definitions["OPENEXR_BUILD_SHARED"] = self.options.shared
+        cmake.definitions["OPENEXR_BUILD_STATIC"] = not bool(self.options.shared)
```

Comment:
```suggestion
        cmake.definitions["OPENEXR_BUILD_STATIC"] = not self.options.shared
```
Don't need to convert again.